### DISCUSSION TITLE
How can I use path segments in `vercel dev`?

### DISCUSSION THREAD

**Discussion by YajTPG** [vercel/vercel#8327] [Help] [answered]
I'm using React with NodeJS for my serverless functions language and [Path Segments](https://vercel.com/docs/concepts/functions/serverless-functions#path-segments) don't seem to work as intended, I get a 404 as defined in my react app.

**Comment by YajTPG** [comment]
Creating a request to `/api/[api]` works, but not `/api/test`.

**Comment by amyegan** [comment, marked as answer]
Hi @YajTPG! Sorry you're running into trouble with Path Segments. I just set up a test app using [create-react-app](https://create-react-app.dev/) and the instructions from the [Path Segments](https://vercel.com/docs/concepts/functions/serverless-functions#path-segments) page you shared, and it seems to be working. There might be something more complex in your project or a misconfiguration. If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look.

My minimal setup (https://github.com/amyegan/path-segments)
<img width="646" alt="image" src="https://user-images.githubusercontent.com/8247105/183111136-50279e0b-2296-43f4-8485-89182c1b4e46.png">
 
The result from my test app (https://path-segments.vercel.app/api/name/YajTPG)
```
Hello YajTPG!
```
This also works on localhost using `vercel dev`.

**Reply by YajTPG** [reply]
For some reason switching the framework to `Create React App` (I was using `Other` before) and using the default commands instead of overrides in Vercel settings fixes it!
Kind of weird since my current dev command `npm run start` runs the same thing. ¯\\_\(ツ)\_\/¯

**Reply by amyegan** [reply]
Very weird, but I'm glad it's working now!
